exporter/elasticsearchexporter/config.go

```diff
@@ -41,18 +41,15 @@ type Config struct {
 	NumWorkers int `mapstructure:"num_workers"`
 
 	// This setting is required when logging pipelines used.
```

Comment:
Is this comment still valid?

Reply:
fixed in [378f81f](https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/38500/commits/378f81fb5755d5f78a2974aafac4c7358e1a23f1)